dll/win32/browseui/desktopipc.cpp

```diff
@@ -366,10 +366,23 @@ static HRESULT ExplorerMessageLoop(IEThreadParamBlock * parameters)
     }
 
     CComPtr<IShellBrowser> psb;
+#if 0 // TODO
+    if (!(parameters->dwFlags & (SH_EXPLORER_CMDLINE_FLAG_E | SH_EXPLORER_CMDLINE_FLAG_NOREUSE)))
```

Comment:
What would be the future purpose of this block?

Reply:
Reuse any existing Explorer window that is already open at that PIDL location. This requires IShellWindows and marshaling of the IDispatch/IWebBrowser object etc.

Reply:
I would suggest you add an explanation for the "TODO" then.

Reply:
> Do you know what the purpose of "SH_EXPLORER_CMDLINE_FLAG_E" is, by the way? (naming doesn't explain that at all)
The flags with a single letter seem to be the Explorer.exe switches, in this case, /E (show Explorer tree band).

> I would suggest you add an explanation for the "TODO" then.

It should be clear enough by the fact that it does not happen if the NOREUSE flag is set and that the unfinished code tries to "call" `IShellWindows::FindWindowSW` which is only ever done to find an existing Explorer window.

Reply:
> The flags with a single letter seem to be the Explorer.exe switches

aha, good point